client-tags/react.md

```diff
@@ -0,0 +1,82 @@
+---
+title: `+react` client tag
+layout: spec
+work-in-progress: true
+copyrights:
+  -
+    name: "James Wheare"
+    period: 2016
+    email: "james@irccloud.com"
+---
+
+## Notes for implementing work-in-progress version
+
+This is a work-in-progress specification.
+
+Software implementing this work-in-progress specification MUST NOT use the
+unprefixed `+react` tag name. Instead, implementations SHOULD use the
+`+draft/react` capability name to be interoperable with other software
+implementing a compatible work-in-progress version.
+
+The final version of the specification will use an unprefixed tag name.
+
+## Introduction
+
+This specification defines a client-only message tag to indicate reactions to other messages
+
+## Motivation
+
+This tag provides a means of communicating with context-sensitive, potentially non-textual reactions. It allows chat participants to respond to each other with text, symbols, emoticons, or emoji that don't necessarily appear as full messages, but instead as lightweight annotations, displayed adjacent to a parent message.
+
+## Architecture
+
+### Dependencies
+
+Clients wishing to use this tag MUST negotiate the [`draft/message-tags`](../core/message-tags-3.3.html) capability with the server. Additionally, this tag MUST be used in conjunction with the [`+draft/reply`](./reply.html) client tag.
+
+### Format
+
+The react tag is sent by a client with the client-only prefix `+`. The value has no restrictions.
+
+    +draft/react=<reaction>
```

Comment:
Shouldn't this be limit to the Unicode Miscellaneous Symbols and Pictographs block?